docs/persistence.rst

```diff
@@ -110,6 +110,31 @@ you have custom functions (say, a custom function to be used with
 most ``numpy`` and ``scipy`` functions should work. Therefore, you can save
 objects having references to functions such as ``numpy.sqrt``.
 
+Command Line Interface
+######################
+
+Skops has a command line interface to convert scikit-learn models persisted with
+``Pickle`` to ``Skops`` files.
+
+To convert a file from the command line, use the ``skops convert`` entrypoint.
+
+Below is an example call to convert a file ``my_model.pkl`` to ``my_model.skops``:
+
+.. code:: console
+
+    skops convert my_model.pkl
+
+To convert multiple files, you can use bash commands to iterate the above call.
+For example, to convert all ``.pkl`` flies in the current directory:
+
+.. code:: console
+
+    for FILE in *.pkl; do skops convert FILE; done
```

Comment:
Just noticed, wouldn't `skops convert *.pkl` do the same?

Reply:
I played around with that syntax, and I think it essentially calls `skops convert file_1.pkl file_2.pkl`, not `skops convert file_1.pkl & skops convert file_2.pkl`

The functionality to do `skops convert file_1.pkl file_2.pkl` was removed following the earlier comments on this PR, but I think we could consider the right way to handle multiple input files at a single time

Reply:
Ah yes, you're right, I misremembered how globbing is resolved (also not sure if Windows works the same as unix with globbing).

It's unfortunate that this doesn't work. I tried to find the minimal change required to make it work and came to this:

```python
# _convert.py
# l 69
- parser_subgroup.add_argument("input", help="Path to an input file to convert. ")
+ parser_subgroup.add_argument("input", help="Path to an input file to convert. ", nargs='+')

# l 95, replace main with
def main(
    parsed_args: argparse.Namespace,
) -> None:
    input_files = parsed_args.input
    output_file = parsed_args.output_file
    if len(input_files) > 1 and output_file:
        raise ValueError("TODO")

    for input_file in input_files:
        logging.basicConfig(format="%(message)s", level=get_log_level(parsed_args.loglevel))

        if not output_file:
            # No filename provided, defaulting to base file path
            file_name = pathlib.Path(input_file).stem
            output_file = pathlib.Path.cwd() / f"{file_name}.skops"

        _convert_file(
            input_file=input_file,
            output_file=output_file,
        )
        output_file = None
```

So basically the same as the old main function, but putting it inside of a loop.

I know this is a bit of a rollback of what we said earlier of not supporting multiple input files, but it is a simplified implementation of the earlier version of supporting multiple files. That said, I would also be totally fine with keeping it as is and using a loop in bash.

Reply:
Yep, similar to what I had before, but I think the complexity comes from that "ValueError" line

The UX for the user is the complex bit there, my previous implementation would basically allow the user to pass in multiple output files too, and resort to the `if not output_file:` logic for the last files if there was a mismatch between the number of input and output files.

Happy to work this back in, but I think this PR should probably stay to a single file implementation (Adrin seemed to want to stick to this for this PR) and discuss this further later :)

Reply:
Yeah, let's keep this in mind for a possible future PR.

> my previous implementation would basically allow the user to pass in multiple output files too

I think that most of the complexity came from exactly that, since it required dealing with num_inputs != num_outputs, while here, we just don't allow specifying outputs for more than 1 input, which is less user friendly but reduces complexity.